src/libclipper/include/clipper/app_metrics.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef CLIPPER_APP_METRICS_HPP
```

Comment:
This has been moved to its own file because it can be used to gather metrics for benchmarking (as is done here https://github.com/nishadsingh1/clipper/blob/benchmark/src/benchmarks/src/noop_bench.cpp#L192).